src/nestjs-back/src/chat/channels/channels.service.ts

```diff
@@ -90,6 +90,14 @@ export class ChannelsService {
 		return channel;
 	}
 
+	async nameIsAvailable(name: string) {
```

Comment:
Is there a specific reason to use query builder? While it's a very powerful feature I would prefer using more simple query forms if not too complex.
For instance, this can be rewritten as:
```ts
const channel = await this.channelsRepository.findOne({
  where: {
    name
  }
})
```
This is simpler

Reply:
Much better, thanks!